activesupport/lib/active_support/cache/memory_store.rb

```diff
@@ -18,6 +18,23 @@ module Cache
     #
     # MemoryStore is thread-safe.
     class MemoryStore < Store
+      module DupCoder
```

Comment:
Should this be `:nodoc:`?

```suggestion
      module DupCoder :nodoc:
```